waiter/waiter.go

```diff
@@ -0,0 +1,42 @@
+package waiter
+
+import (
+	"time"
+)
+
+var (
+	defaultPollInterval = 1 * time.Second
+)
+
+// Executor describes requirements for async.
+type Executor interface {
+	ContainerDeleteExecutor
+	ContainerSetEACLExecutor
+	ContainerPutExecutor
```

Comment:
It's interesting that in fact we only have a small set of container operations that need this.